src/components/pod/pod.d.ts

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+export interface PodProps {
```

Comment:
Ah when I've looked into this again following Ian's comment, these props are all optional so need a `?`

Reply:
good point!